src/main/java/at/hannibal2/skyhanni/features/garden/visitor/GardenVisitorFeatures.kt

```diff
@@ -457,6 +458,10 @@ class GardenVisitorFeatures {
         if (GardenAPI.inGarden() && config.hideChat && hideVisitorMessage(event.message)) {
             event.blockedReason = "garden_visitor_message"
         }
+
+        partialAcceptedPattern.matchMatcher(event.message) {
```

Comment:
Only show the chat message when the skyhanni shopping list is enabled.

Reply:
Done

Reply:
We also don't need to check the pattern if the setting is disabled.

Reply:
Well not relevant enough ig